TextField/TextField.jsx

```diff
@@ -74,6 +74,9 @@ class TextFieldInput extends MaterialComponent {
   componentWillUnmount() {
     this.MDComponent && this.MDComponent.destroy && this.MDComponent.destroy();
   }
+  value() {
```

Comment:
Wouldn't it be better to name it either `getValue()` or make it a property with a getter method?

Reply:
a value prop is already supported to be mapped to native element but yes making it `getValue`